README.md

```diff
@@ -63,3 +63,23 @@ This is not an official Google product.
 This library is in alpha and will be going through a lot of breaking changes. While releases will be stable enough for prototyping, experimentation and research, we do not recommend using it in any production environment yet.
 Expect bugs and sharp edges.
 Please help by trying it out, reporting bugs, and letting us know what you think!
+
+# TX Power Control for nrf52840 dongles
```

Comment:
Maybe this doc should live in a section under `docs/mkdocs/src` (maybe `platforms/zephyr.md`?) rather than the top-level README, which should remain very generic.